src_kt/javatests/arcs/storage/BUILD

```diff
@@ -0,0 +1,18 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_test")
+
+[
```

Comment:
is there a kt_js_test target? Perhaps even if there's no equivalent JUnit for JS for Kotlin, perhaps just a compile test would be useful to insure no JS breaking dependencies were introduced.

Reply:
the travis command will build _every_ target in `java` and `javatests`, which means it will build `kt_js_library` targets as well, so I don't think we need to worry about `kt_js_test`.